packages/language-server/src/plugins/PluginHost.ts

```diff
@@ -400,6 +402,20 @@ export class PluginHost implements LSProvider, OnWatchFileChanges {
         }
     }
 
+    async getSemanticTokens(textDocument: TextDocumentIdentifier, range?: Range) {
+        const document = this.getDocument(textDocument.uri);
+        if (!document) {
+            throw new Error('Cannot call methods on an unopened document');
+        }
+
+        return this.execute<SemanticTokens>(
+            'getSemanticTokens',
+            [document, range],
+            ExecuteMode.Collect,
+            collectSemanticTokens
```

Comment:
This can be removed and changed to `ExecuteMode.FirstNotNull` if we decide not to include svelte semantic tokens.

Reply:
What do you mean by "Svelte semantic tokens"? I don't see another plugin providing tokens besides the TS plugin, so yeah we could change this to `FirstNonNull`.

Reply:
Yeah. I was initially thinking maybe we could have a svelte specific one but can't think of any. I'll just remove it. And if anyone got any idea of svelte specific semantic tokens we can do it in another PR